package.json

```diff
@@ -25,7 +25,13 @@
         "test": "yarn test-plugin && yarn test-app",
         "coverage": "yarn test-plugin --coverage && yarn test-app --coverage",
         "validate-commit": "d2-style js check --staged && d2-style text check --staged",
-        "validate-push": "yarn test"
+        "validate-push": "yarn test",
+        "cypress:start": "BROWSER=none yarn start",
+        "cy:open-live": "cypress_dhis2_api_stub_mode=DISABLED d2-utils-cypress open --appStart 'yarn cypress:start'",
```

Comment:
The test set up doesn't look at the `cypress_dhis2_api_stub_mode`, right? Then these can be removed from the scripts. Also, `cy:capture` can be removed completely.

Reply:
Removed and renamed the remaining two scripts to just `cy:open` and `cy:run`